src/export/Package.ts

```diff
@@ -47,9 +67,25 @@ export class Package implements Fishable {
           break;
         case Type.ValueSet:
           def = this.valueSets.find(vs => vs.id === item || vs.name === item || vs.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'ValueSet'
```

Comment:
Again, under the assumption that _most_ instances are not ValueSets, it might be more efficient to check `i.resourceType === 'VaueSet'` first.